servicemanager/service/smplayservice.py

```diff
@@ -117,24 +119,38 @@ def start_from_binary(self):
                 print b.fail + "ERROR: could not start '" + self.service_name + "' " + b.endc
             return popen_output.pid
 
-    def _unzip_play_application(self):
+    def _unpack_play_application(self, extension):
```

Comment:
Yes, we need to work out what to do based on the extension but my point is that we can infer the extension from the filename.

i.e. `get_play_app_extension ` could be
```
filename, extension = os.path.splitext(filename)
return extension
```